src/Blockcore.Indexer.Core/Operations/SlowRequestsFilteerAttribute.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+using ConcurrentCollections;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace Blockcore.Indexer.Core.Operations;
+
+[AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+public class SlowRequestsFilteerAttribute : ActionFilterAttribute
```

Comment:
We need to discuss this logic.
While I see the point it will mean the node will handle one call per contract.
Surely we can do better.